ui/helpers/constants/common.ts

```diff
@@ -23,3 +23,24 @@ export const OUTDATED_BROWSER_VERSIONS = {
   // See https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser
```

Comment:
Probably we could treat Opera 99-(107?) as affected as well, but maybe not worth worrying about given the impact of the bug. We can adjust this later if we need to. Who knows what they're changing in their fork.